tools/parallel-lint.sh

```diff
@@ -27,6 +27,10 @@ if $ALL; then
 	if php -r 'exit( PHP_VERSION_ID < 70000 ? 0 : 1 );'; then
 		SKIPS+=( -o -name php7 )
 	fi
+	if php -r 'exit( PHP_VERSION_ID < 70200 ? 0 : 1 )'; then
```

Comment:
Oh, there's a typo in here too.
```suggestion
	if php -r 'exit( PHP_VERSION_ID < 70200 ? 0 : 1 );'; then
```